pages/docs/manual/latest/record.mdx

```diff
@@ -189,6 +285,112 @@ getAge(me) // type error!
 
 The type system will complain that `me` is a `person`, and that `getAge` only works on `monster`. If you need such capability, use ReScript objects, described [here](object.md).
 
+### Optional Fields in Records Can Be Useful for Bindings
+Many JavaScript APIs tend to have large configuration objects that can be a bit annoying to model as records, since you previously always needed to specify all record fields when creating a record. 
+
+Optional record fields, introduced in [`v10`](/blog/release-10-0-0#experimental-optional-record-fields), is intended to help with this. Optional fields will let you avoid having to specify all fields, and let you just specify the one's you care about. A significant improvement in ergonomics for bindings and other APIs with for example large configuration objects.
+
+### Pattern Matching on Optional Fields
+[Pattern matching](pattern-matching-destructuring), one of ReScript's most important features, has two caveats when you deal with optional fields.
+
+When matching on the field directly, the value of the field is always treated as `option`. Example:
```

Comment:
The example is illustrative, but it does not really matter that the value you're matching comes from field access.
Perhaps change the phrasing so that this example is the old way, and the next 2 examples are the new way?